routes/api/v1/favorites.js

```diff
@@ -37,4 +41,36 @@ async function favoriteCities(user) {
   }
 }
 
+async function getCoordinates(cityState) {
+  try {
+    let response = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${cityState}&key=${process.env.GOOGLE_API_KEY}`);
+    let data = await response.json();
+    let coordinates = data.results[0].geometry.location;
+    return coordinates;
+  }catch(e) {
+    return e;
+  }
+}
+
+async function getForecast(coordinates) {
+  try{
+    let response = await fetch(`https://api.darksky.net/forecast/${process.env.DARK_SKY_API_KEY}/${coordinates.lat},${coordinates.lng}?exclude=minutely,alerts,flags`);
+    let data = await response.json();
+    return data;
+  }catch(e){
+    return e;
+  }
+}
+
+const forecastFor = (cities) => {
+  const promises = cities.map(async (city) => {
+    let coordinates = await getCoordinates(city.location)
+      return {
+          location: `${city.location}`,
+          forecast: await getForecast(coordinates)
```

Comment:
This looks good!  really small thing but if you want to be consistent with the rubric maybe change "forecast" to "current_weather"  on line70